sensor/common/networkflow/manager/manager_impl.go

```diff
@@ -190,18 +231,23 @@ func (m *networkFlowManager) Capabilities() []centralsensor.SensorCapability {
 }
 
 func (m *networkFlowManager) ResponsesC() <-chan *central.MsgFromSensor {
-	return m.flowUpdates
+	return m.sensorUpdates
 }
 
 func (m *networkFlowManager) enrichConnections() {
```

Comment:
Shouldn't we have a new enrichEndpoints function instead of using enrichConnections()?

Reply:
I don't really see a need to create a completely separate goroutine to send the data since:
- We are using the same data available in the existing endpoints.
- We are using the same channel to send messages.

The only real difference is in how we process the endpoint data to send over to central.

Reply:
Okay